src/ui-services/notificationsService.js

```diff
@@ -21,6 +21,8 @@ angular.module('openshiftCommonUI').provider('NotificationsService', function()
       // notifications may already have an id that is not necessarily unique,
       // this is an explicitly unique id just for `track by` in templates
       notification.trackByID = _.uniqueId('notification-') + Date.now();
+      notification.skipToast = notification.skipToast || false;
+      notification.showInDrawer = notification.showInDrawer || false;
```

Comment:
I would think ```showInDrawer``` would default to ```true```?

Reply:
The notifications are formatted very differently from events (icons, titles, and messages), so it can look strange to mix them. For example:

![screen shot 2017-09-06 at 1 27 35 pm](https://user-images.githubusercontent.com/1167259/30125676-31fa0d16-9307-11e7-899d-b0478d9f9d5f.png)

Reply:
Yeah, I wonder if we need to clarify hierarchy.  The ideal would be that the events page retains ALL history (events + notifications), the drawer a curated history (important events + notifications), and toasts be less frequent but timely notifications (and ideally not notifying a user of something they already know, ie, just clicked the "build" button).  This has a few problems in that TTL for notifications and TTL for events is entirely different.  We don't store notifications at all, but events have a 2hr TTL from the server.   We also don't include any notification information on the events list page.

@dtaylor113 we are really hashing this out in discussion [here](https://github.com/openshift/origin-web-console/pull/2001#issuecomment-327587103) and [here](https://github.com/openshift/origin-web-console/pull/2001#issuecomment-327790513) (and everything in between).  The problem is that we are a excessively pestering the user sometimes.  

We should probably keep in mind the "mini-toast" idea, would we need an extra config option here? (suggested mini toast would be like gmail notifications):

![screen shot 2017-09-06 at 3 16 27 pm](https://user-images.githubusercontent.com/280512/30130220-792e4ce2-9316-11e7-9413-e344961aefaa.png)

Thoughts?